be/src/http/action/stream_load.cpp

```diff
@@ -705,8 +706,15 @@ Status StreamLoadAction::_data_saved_path(HttpRequest* req, std::string* file_pa
 
 void StreamLoadAction::_save_stream_load_record(std::shared_ptr<StreamLoadContext> ctx,
                                                 const std::string& str) {
-    auto stream_load_recorder =
-            ExecEnv::GetInstance()->storage_engine().to_local().get_stream_load_recorder();
+    std::shared_ptr<StreamLoadRecorder> stream_load_recorder;
+    if (config::is_cloud_mode()) {
+        stream_load_recorder =
+                ExecEnv::GetInstance()->storage_engine().to_cloud().get_stream_load_recorder();
```

Comment:
The get_stream_load_recorder has been implemented in the base class, to_cloud or to_local is not necessary？